debezium-connector-ibmi/src/main/java/io/debezium/connector/db2as400/As400ConnectorConfig.java

```diff
@@ -57,33 +57,35 @@ public class As400ConnectorConfig extends RelationalDatabaseConnectorConfig {
     /**
      * keep alive flag, should the driver send keep alive packets default true
      */
-    public static final Field KEEP_ALIVE = Field.create("keep.alive", "keep alive",
+    public static final Field KEEP_ALIVE = Field.create(DATABASE_CONFIG_PREFIX + "keep alive", "keep alive",
             "keep alive", true);
 
     /**
      * keep alive flag, should the driver use a secure connection defaults to false
      */
-    public static final Field SECURE = Field.create("secure", "secure", "use secure connection", true);
+    public static final Field SECURE = Field.create(DATABASE_CONFIG_PREFIX + "secure", "secure", "use secure connection", true);
```

Comment:
Recently it was reported that to make `secure=false` work it is necessary to set both `secure=false` and `database.secure=false`. Will this change solve the issue?

Reply:
I believe it does, I'm testing locally with secure=false and it's working with just the one setting

Reply:
I've tested with both true and false and both the RPC connection and JDBC both change with the one setting